### DISCUSSION TITLE
Pipe node dynamic slot name

### DISCUSSION THREAD

**Discussion by MMoneer** [crystian/ComfyUI-Crystools#46] [Ideas]
Hello,
I'd suggest making slots of pipe nodes that take the name from the input slot and pass it to the output slot
If the input link from the model slot changes any1 slot name to model and passes it to the output pipe any1 output name changes to  model.
<img width="600" alt="pipe-1" src="https://github.com/crystian/ComfyUI-Crystools/assets/18519387/04ae6adc-b197-4a3e-8d5c-7a82e4b5cd8d">
Thank you

**Comment by crystian** [comment]
I like to do it, but I don't have time sorry :(

**Reply by MMoneer** [reply]
Thank you for your reply and your effort on this project.
